catalogs/catalog-lakehouse-iceberg/src/main/java/org/apache/gravitino/catalog/lakehouse/iceberg/IcebergCatalogPropertiesMetadata.java

```diff
@@ -98,7 +72,10 @@ public class IcebergCatalogPropertiesMetadata extends BaseCatalogPropertiesMetad
                 false),
             stringRequiredPropertyEntry(URI, "Iceberg catalog uri config", false, false),
             stringRequiredPropertyEntry(
-                WAREHOUSE, "Iceberg catalog warehouse config", false, false));
+                WAREHOUSE, "Iceberg catalog warehouse config", false, false),
+            stringOptionalPropertyEntry(IcebergConstants.GRAVITINO_S3_ACCESS_KEY_ID, "s3 access-key-id", true, null, true),
```

Comment:
make it hidden

Reply:
If we make it hidden, the engine side couldn't load it automatically, and should set the access key and secret key explictly.